core/src/main/resources/AppConstants.properties

```diff
@@ -103,6 +103,13 @@ transactionmanager.narayana.stuckTime=180000
 # How often (in ms) the connection pool checks for stuck connections.
 transactionmanager.narayana.stuckTimerTime=30000
 
+## Narayana ObjectStore Settings
+# When using the database the FQDN must be used, eg; com.arjuna.ats.internal.arjuna.objectstore.jdbc.JDBCStore
+transactionmanager.narayana.objectStoreType=com.arjuna.ats.internal.arjuna.objectstore.ShadowNoFileLockStore
+transactionmanager.narayana.objectStoreDatasource=
```

Comment:
Kan je hier ook wat documentatie bij zetten? Wat hier in zou moeten staan: JDBC URL?; dat de datasource non-XA moet zijn. Wat van dat soort dingen die je als gebruiker moet weten.